### PR TITLE
MCKIN-7712: revert incorrect tile to drag items container on popup close

### PR DESCRIPTION
**Pull request by msaqib52**
Incorrect Placed Tile Goes Back to its Placeholder when close button is clicked in the pop up window.

### CODE REVIEW

**Comment by msaqib52** [comment]
@pomegranited this PR is ready for review.

File: drag_and_drop_v2/public/js/drag_and_drop.js [hunk 1]
```diff
@@ -1686,15 +1686,13 @@ function DragAndDropBlock(runtime, element, configuration) {
         $.post(url, JSON.stringify(data), 'json')
             .done(function(data){
                 state.items[item_id].submitting_location = false;
-                // In standard mode we immediately return item to the bank if dropped on wrong zone.
+                // In standard mode we return wrongly dropped item to the bank when feedback popup is closed.
                 // In assessment mode we leave it in the chosen zone until explicit answer submission.
                 if (configuration.mode === DragAndDropBlock.STANDARD_MODE) {
```
Comment (pomegranited) [inline drag_and_drop_v2/public/js/drag_and_drop.js]: This feature would also be useful in assessment mode, but I think this is out of scope for this change?

Comment (msaqib52) [reply drag_and_drop_v2/public/js/drag_and_drop.js]: yes there would be separate ticket for assessment mode, if needed.

File: tests/integration/test_interaction.py [hunk 2]
```diff
@@ -125,10 +125,10 @@ def parameterized_item_negative_feedback_on_bad_move_standard(
                 self.wait_until_html_in(definition.feedback_negative, feedback_popup_content)
                 self.assert_popup_incorrect(popup)
                 self.assertTrue(popup.is_displayed())
-                self.assert_reverted_item(definition.item_id)
```
[the remark this answers is not in the document]
Comment (msaqib52) [reply tests/integration/test_interaction.py]: done

File: tests/integration/test_interaction.py [hunk 3]
```diff
@@ -419,14 +419,14 @@ def test_keyboard_drag_zones_outline(self):
                 self.assertNotIn("dragging", drag_container.get_attribute('class').split(' '))
 
     def test_mouse_drag_zones_outline(self):
-        self.scroll_down(pixels=200)
+        self.scroll_down(pixels=300)
```
[the remark this answers is not in the document]
Comment (msaqib52) [reply tests/integration/test_interaction.py]: yup this is strange, without this tests were failing in this PR.

**Comment by msaqib52** [comment]
@pomegranited Thank you for the review, i have updated test with suggested changes.
